Dockerfile

```diff
@@ -1,44 +1,5 @@
-FROM ocaml/opam2:alpine-3.10-ocaml-4.10 AS builder
+FROM docker.pkg.github.com/jtcoolen/bot/coqbot:latest
```

Comment:
Instead of pushing the base image to a registry to be able to reuse it on this line, is there a way for this `FROM` directive to use some local image?

Reply:
Well we could execute a script within our deploy action which would build the release.Dockerfile and tag it (`docker build -f Dockerfile . -t coqbot`), then build our Dockerfile beginning with `FROM coqbot`. The current deploy action looks for a Dockerfile in the repository to build, but I'm not sure if it would find the tagged coqbot image that was just built.

Reply:
It turns out that this works pretty well this way.